File: VisitBosnia/VisitBosnia.Model/SearchObjects/ForumSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.SearchObjects
{
    public class ForumSearchObject
    {
        //public string? SearchText { get; set; }
        public string? Title { get; set; }
        public int? CityId { get; set; }
        public bool? IncludeCity { get; set; }

    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/forum/forum_search_object.dart
class ForumSearchObject {
  String? title;
  int? cityId;
  bool? includeCity;

  ForumSearchObject({this.title, this.cityId, this.includeCity});

  ForumSearchObject.fromJson(Map<String, dynamic> json) {
    title = json['title'];
    cityId = json['cityId'];
    includeCity = json['includeCity'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['title'] = this.title;
    data['cityId'] = this.cityId;
    data['includeCity'] = this.includeCity;
    return data;
  }
}
